packages/itwinui-react/src/core/Alert/Alert.tsx

```diff
@@ -37,73 +147,106 @@ export type AlertProps = {
   children: React.ReactNode;
 } & Omit<CommonProps, 'title'>;
 
+export const AlertComponent = React.forwardRef(
```

Comment:
- this AlertComponent should not be exported
- should be polymorphic
- should be moved to the top of the file (preference)

Reply:
These all have been changed

Reply:
when defining polymorphic components, there is no need for explicitly adding `children` or `CommonProps`, so those can be removed from `AlertOwnProps`

Reply:
Removed